### PR TITLE
fix kublet dir permissions

### PR DESCRIPTION
**Pull request by glitchcrab**
towards https://github.com/giantswarm/giantswarm/issues/25275

This PR:

- fix kubelet dir permissions so node-exporter works correctly

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/cluster-cloud-director/templates/_helpers.tpl [hunk 1]
```diff
@@ -135,16 +135,6 @@ files:
 {{- include "staticRoutes" . | nindent 2}}
 {{- end }}
 
-preKubeadmCommands:
```
Comment (vxav) [inline helm/cluster-cloud-director/templates/_helpers.tpl:138]: Why remove these?

Comment (glitchcrab) [reply helm/cluster-cloud-director/templates/_helpers.tpl:138]: They're not used, the same commands are already in the template.

Comment (vxav) [reply helm/cluster-cloud-director/templates/_helpers.tpl:138]: They're in [kubeadmcontrolplane](https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml#L125) but not in [kubeadmconfigtemplate](https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/templates/kubeadmconfigtemplate.yaml#L13)

**Comment by glitchcrab** [comment]
whatamess

**Pull request closed without merging**